### DISCUSSION TITLE
Why is the project called "satori"?

### DISCUSSION THREAD

**Discussion by hmmhmmhm** [vercel/satori#176] [General]
Is that what I like?

**Comment by shuding** [comment]
Because when creating the project I was playing Zelda Breath of the Wild and there's a beautiful mountain called Satori in it: https://zelda.fandom.com/wiki/Satori_Mountain

And I thought it's a good name 😄

**Reply by hmmhmmhm** [reply]
Thx so much for your answer!! I wanted to know the origin because it was an interesting name!

**Reply by anaclumos** [reply]
Haha that's cute!

**Comment by styfle** [comment]
- [satori](https://github.com/vercel/satori) starts with "S"
- [swr](https://github.com/vercel/swr) starts with "S"
- [styled-jsx](https://github.com/vercel/styled-jsx) starts with "S"
- [serve](https://github.com/vercel/serve) starts with "S"
- @shuding starts with "S"
- @styfle starts with "S"

I think I see a pattern 🕵️ 😄

**Reply by shuding** [reply]
Yeah that's the <b>s</b>ecret behind it!

**Comment by Jackie1210** [comment]
2333
